api/candlepin-api-spec.yaml

```diff
@@ -2432,55 +2735,1142 @@ paths:
-        404:
-          description: Owner key could not be found
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/ExceptionMessage'
-              example:
-                displayMessage: Owner key could not be found
-                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
-        default:
-          $ref: '#/components/responses/default'
-
-    post:
-      description: Creates a product for an owner
-      tags:
+        - name: uuid
+          in: query
+          description: The UUID of consumers
+          schema:
+            type: array
+            items:
+              type: string
+        - name: hypervisor_id
+          in: query
+          description: The hypervisor IDs
+          schema:
+            type: array
+            items:
+              type: string
+        - name: fact
+          in: query
+          description: The consumer facts
+          schema:
+            type: array
+            items:
+              $ref: '#/components/schemas/KeyValueParamDTO'
+        - name: sku
+          in: query
+          description: The list of skus
+          schema:
+            type: array
+            items:
+              type: string
+        - name: subscription_id
+          in: query
+          description: The list of subscription IDs
+          schema:
+            type: array
+            items:
+              type: string
+        - name: contract
+          in: query
+          description: The list of contract numbers
+          schema:
+            type: array
+            items:
+              type: string
+      security: []
+      responses:
+        200:
+          description: Consumers successfully retrieved
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ConsumerDTOArrayElement'
+        400:
+          description: Owner key is null or empty
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+              example:
+                displayMessage: Owner key is null or empty
+                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
+        404:
+          description: An owner could not be found using the provided key
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+              example:
+                displayMessage: An owner could not be found using the provided key
+                requestUuid: c4347004-8792-41fe-a4d8-fccaa0d3898a
+        default:
+          $ref: '#/components/responses/default'
+
+  /owners/{owner_key}/consumers/count:
+    get:
+      description: Retrieves a count of consumers for the owner
+      tags:
+        - Owner
+        - Consumer
+      operationId: countConsumers
+      parameters:
```

Comment:
Parameters descriptions are incorrect at multiple places in this endpoint.

Reply:
Added descriptions.